### ISSUE TITLE
Website autoupdate is currently broken

### ISSUE THREAD

**Issue by aaruni96** [oscar-system/oscar-website#394]
Merging PR #386 broke the website's auto update

My comment from the PR :

> So, this did break the web server...
> ```
> update.sh[1962771]: You have requested:
> update.sh[1962771]:   sassc = 2.1.0
> update.sh[1962771]: The bundle currently has sassc locked at 2.4.0.
> update.sh[1962771]: Try running `bundle update sassc`
> update.sh[1962771]: If you are updating multiple gems in your Gemfile at once,
> update.sh[1962771]: try passing them all to `bundle update`
> systemd[1317]: oscar-website.service: Main process exited, code=exited, status=7/NOTRUNNING
> systemd[1317]: oscar-website.service: Failed with result 'exit-code'.
> systemd[1317]: Failed to start "Update OSCAR website".
>```
>
>Should I try running the bundle update manually? (In theory this should be a one time step, as either the bundle update will >update Gemfile.lock, or we can manually remove the lockfile, and run bundle install. In any case, the new lockfile will have >sassc=2.1.0)

Thoughts @fingolfin  ?

**Comment by fingolfin** [comment]
Resolved the reported issue by `rm data/oscar-website/Gemfile.lock && touch data/oscar-website.trigger` 

But now it runs into a new issue:
```
Dec 04 19:47:37 www-admin11 update.sh[1997512]: Cloning into 'Hecke.jl'...
Dec 04 19:48:20 www-admin11 update.sh[1997512]: [2.0K blob data]
Dec 04 19:48:20 www-admin11 update.sh[1997509]: -------------------------------
Dec 04 19:48:20 www-admin11 update.sh[1997509]: Processing thofma/Hecke.jl...
Dec 04 19:48:20 www-admin11 update.sh[1997509]: -------------------------------
Dec 04 19:48:20 www-admin11 update.sh[1997509]: Fetching updates...
Dec 04 19:48:20 www-admin11 update.sh[1997509]: Generating list of authors active in past year...
Dec 04 19:48:20 www-admin11 update.sh[1997509]: Item 1 of 19...
Dec 04 19:48:20 www-admin11 update.sh[1997509]: ['Benjamin Lorenz', 'benlorenz@users.noreply.github.com']
Dec 04 19:48:20 www-admin11 update.sh[1997509]: 401
Dec 04 19:48:20 www-admin11 update.sh[1997509]: <Response [401]>
Dec 04 19:48:20 www-admin11 update.sh[1997509]: {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest","status":"401"}
Dec 04 19:48:20 www-admin11 update.sh[1997509]: benlorenz@users.noreply.github.com
Dec 04 19:48:20 www-admin11 update.sh[1997509]: https://api.github.com/repos/thofma/Hecke.jl/commits/b671e6ffcdfcb6c1a6d87ea611ffdca0af094d10
Dec 04 19:48:20 www-admin11 update.sh[1997509]: Traceback (most recent call last):
Dec 04 19:48:20 www-admin11 update.sh[1997509]:   File "/srv/www/www-mathe-oscar/data/oscar-website/etc/update-contributors.py", line 87, in <module>
Dec 04 19:48:20 www-admin11 update.sh[1997509]:     assert github_username != "__notfound__"
Dec 04 19:48:20 www-admin11 update.sh[1997509]: AssertionError
Dec 04 19:48:20 www-admin11 systemd[1317]: oscar-website.service: Main process exited, code=exited, status=1/FAILURE
Dec 04 19:48:20 www-admin11 systemd[1317]: oscar-website.service: Failed with result 'exit-code'.
Dec 04 19:48:20 www-admin11 systemd[1317]: Failed to start "Update OSCAR website".
Dec 04 19:48:20 www-admin11 systemd[1317]: oscar-website.service: Consumed 36.837s CPU time.
```


Why exactly is it cloning `Hecke.jl` ?!

**Comment by fingolfin** [comment]
Should be resolved by PR #396, @aaruni96
